cypress/apiRequests/Product.js

```diff
@@ -1,13 +1,12 @@
 import { getValueWithDefault } from "./utils/Utils";
 
-class Product {
-  getFirstProducts(first, search) {
-    const filter = search
-      ? `, filter:{
+export function getFirstProducts(first, search) {
```

Comment:
When we are at refactoring - can we use gql tag like in queries in dashboard?

Reply:
Investigated it a bit and more changes in making requests would be needed. It's a material for separate PR